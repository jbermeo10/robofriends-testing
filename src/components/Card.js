import React from 'react';
import { shallow, mount, render } from 'enzyme';

//const Card = (props) => {
const Card = ({ id, name, email }) => {
  //const { name, email, id } = props;
  return (
    //<h1>Robo Friends</h1>
    <div className='tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5'>
      <img alt='robots' src={`https://robohash.org/${id}`} width={200} height={200}/>
      <div>
        <h2>{name}</h2>
        <p>{email}</p>
      </div>
    </div>
  );
}

export default Card;
