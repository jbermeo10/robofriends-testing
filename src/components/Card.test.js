import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card';

// console.log(shallow(<Card />))
it('se espera renderizar el componente Card', () => {
  // expect(shallow(<Card />).length).toEqual(1);
  expect(shallow(<Card />)).toMatchSnapshot();
})

