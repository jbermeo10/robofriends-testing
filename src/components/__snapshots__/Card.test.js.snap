// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`se espera renderizar el componente Card 1`] = `
<div
  className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robots"
    height={200}
    src="https://robohash.org/undefined"
    width={200}
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
